.gitignore

```diff
@@ -13,6 +13,8 @@ bin-release/
 *.ipa
 *.apk
 
+test.md
+
 # Project files, i.e. `.project`, `.actionScriptProperties` and `.flexProperties`
 # should NOT be excluded as they contain compiler settings and other important
 # information for Eclipse / Flash Builder.
```

Comment:
- [ ] OPTIONAL: Please, kindly remove commented code from .gitignore, because it makes your project not clean.